src/frontend/components/actions/new.tsx

```diff
@@ -87,7 +87,7 @@ const New: FC<ActionProps> = (props) => {
               record={record as RecordJSON}
             />
           ))
-          : resource.editProperties.map((property) => (
+          : resource.newProperties.map((property) => (
```

Comment:
We might also need to update `where` to `new` in `BasePropertyComponent` and `LayoutElementRenderer`. `new` would then also have to be added to i. e. `isVisible` for properties etc.

Reply:
done too